%  LaTeX support: latex@mdpi.com 
%  For support, please attach all files needed for compiling as well as the log file, and specify your operating system, LaTeX version, and LaTeX editor.

%=================================================================
\documentclass[biotech,article,submit,pdftex,moreauthors]{Definitions/mdpi} 

%--------------------
% Class Options:
%--------------------
%----------
% journal
%----------
% Choose between the following MDPI journals:
% acoustics, actuators, addictions, admsci, adolescents, aerobiology, aerospace, agriculture, agriengineering, agrochemicals, agronomy, ai, air, algorithms, allergies, alloys, analytica, analytics, anatomia, animals, antibiotics, antibodies, antioxidants, applbiosci, appliedchem, appliedmath, applmech, applmicrobiol, applnano, applsci, aquacj, architecture, arm, arthropoda, arts, asc, asi, astronomy, atmosphere, atoms, audiolres, automation, axioms, bacteria, batteries, bdcc, behavsci, beverages, biochem, bioengineering, biologics, biology, biomass, biomechanics, biomed, biomedicines, biomedinformatics, biomimetics, biomolecules, biophysica, biosensors, biotech, birds, bloods, blsf, brainsci, breath, buildings, businesses, cancers, carbon, cardiogenetics, catalysts, cells, ceramics, challenges, chemengineering, chemistry, chemosensors, chemproc, children, chips, cimb, civileng, cleantechnol, climate, clinpract, clockssleep, cmd, coasts, coatings, colloids, colorants, commodities, compounds, computation, computers, condensedmatter, conservation, constrmater, cosmetics, covid, crops, cryptography, crystals, csmf, ctn, curroncol, cyber, dairy, data, ddc, dentistry, dermato, dermatopathology, designs, devices, diabetology, diagnostics, dietetics, digital, disabilities, diseases, diversity, dna, drones, dynamics, earth, ebj, ecologies, econometrics, economies, education, ejihpe, electricity, electrochem, electronicmat, electronics, encyclopedia, endocrines, energies, eng, engproc, entomology, entropy, environments, environsciproc, epidemiologia, epigenomes, est, fermentation, fibers, fintech, fire, fishes, fluids, foods, forecasting, forensicsci, forests, foundations, fractalfract, fuels, future, futureinternet, futurepharmacol, futurephys, futuretransp, galaxies, games, gases, gastroent, gastrointestdisord, gels, genealogy, genes, geographies, geohazards, geomatics, geosciences, geotechnics, geriatrics, grasses, gucdd, hazardousmatters, healthcare, hearts, hemato, hematolrep, heritage, higheredu, highthroughput, histories, horticulturae, hospitals, humanities, humans, hydrobiology, hydrogen, hydrology, hygiene, idr, ijerph, ijfs, ijgi, ijms, ijns, ijpb, ijtm, ijtpp, ime, immuno, informatics, information, infrastructures, inorganics, insects, instruments, inventions, iot, j, jal, jcdd, jcm, jcp, jcs, jcto, jdb, jeta, jfb, jfmk, jimaging, jintelligence, jlpea, jmmp, jmp, jmse, jne, jnt, jof, joitmc, jor, journalmedia, jox, jpm, jrfm, jsan, jtaer, jvd, jzbg, kidneydial, kinasesphosphatases, knowledge, land, languages, laws, life, liquids, literature, livers, logics, logistics, lubricants, lymphatics, machines, macromol, magnetism, magnetochemistry, make, marinedrugs, materials, materproc, mathematics, mca, measurements, medicina, medicines, medsci, membranes, merits, metabolites, metals, meteorology, methane, metrology, micro, microarrays, microbiolres, micromachines, microorganisms, microplastics, minerals, mining, modelling, molbank, molecules, mps, msf, mti, muscles, nanoenergyadv, nanomanufacturing,\gdef\@continuouspages{yes}} nanomaterials, ncrna, ndt, network, neuroglia, neurolint, neurosci, nitrogen, notspecified, %%nri, nursrep, nutraceuticals, nutrients, obesities, oceans, ohbm, onco, %oncopathology, optics, oral, organics, organoids, osteology, oxygen, parasites, parasitologia, particles, pathogens, pathophysiology, pediatrrep, pharmaceuticals, pharmaceutics, pharmacoepidemiology,\gdef\@ISSN{2813-0618}\gdef\@continuous pharmacy, philosophies, photochem, photonics, phycology, physchem, physics, physiologia, plants, plasma, platforms, pollutants, polymers, polysaccharides, poultry, powders, preprints, proceedings, processes, prosthesis, proteomes, psf, psych, psychiatryint, psychoactives, publications, quantumrep, quaternary, qubs, radiation, reactions, receptors, recycling, regeneration, religions, remotesensing, reports, reprodmed, resources, rheumato, risks, robotics, ruminants, safety, sci, scipharm, sclerosis, seeds, sensors, separations, sexes, signals, sinusitis, skins, smartcities, sna, societies, socsci, software, soilsystems, solar, solids, spectroscj, sports, standards, stats, std, stresses, surfaces, surgeries, suschem, sustainability, symmetry, synbio, systems, targets, taxonomy, technologies, telecom, test, textiles, thalassrep, thermo, tomography, tourismhosp, toxics, toxins, transplantology, transportation, traumacare, traumas, tropicalmed, universe, urbansci, uro, vaccines, vehicles, venereology, vetsci, vibration, virtualworlds, viruses, vision, waste, water, wem, wevj, wind, women, world, youth, zoonoticdis 
% For posting an early version of this manuscript as a preprint, you may use "preprints" as the journal. Changing "submit" to "accept" before posting will remove line numbers.

%---------
% article
%---------
% The default type of manuscript is "article", but can be replaced by: 
% abstract, addendum, article, book, bookreview, briefreport, casereport, comment, commentary, communication, conferenceproceedings, correction, conferencereport, entry, expressionofconcern, extendedabstract, datadescriptor, editorial, essay, erratum, hypothesis, interestingimage, obituary, opinion, projectreport, reply, retraction, review, perspective, protocol, shortnote, studyprotocol, systematicreview, supfile, technicalnote, viewpoint, guidelines, registeredreport, tutorial
% supfile = supplementary materials

%----------
% submit
%----------
% The class option "submit" will be changed to "accept" by the Editorial Office when the paper is accepted. This will only make changes to the frontpage (e.g., the logo of the journal will get visible), the headings, and the copyright information. Also, line numbering will be removed. Journal info and pagination for accepted papers will also be assigned by the Editorial Office.

%------------------
% moreauthors
%------------------
% If there is only one author the class option oneauthor should be used. Otherwise use the class option moreauthors.

%---------
% pdftex
%---------
% The option pdftex is for use with pdfLaTeX. Remove "pdftex" for (1) compiling with LaTeX & dvi2pdf (if eps figures are used) or for (2) compiling with XeLaTeX.

%=================================================================
% MDPI internal commands - do not modify
\firstpage{1} 
\makeatletter 
\setcounter{page}{\@firstpage} 
\makeatother
\pubvolume{1}
\issuenum{1}
\articlenumber{0}
\pubyear{2024}
\copyrightyear{2024}
%\externaleditor{Academic Editor: Firstname Lastname}
\datereceived{ } 
\daterevised{ } % Comment out if no revised date
\dateaccepted{ } 
\datepublished{ } 
%\datecorrected{} % For corrected papers: "Corrected: XXX" date in the original paper.
%\dateretracted{} % For corrected papers: "Retracted: XXX" date in the original paper.
\hreflink{https://doi.org/} % If needed use \linebreak
%\doinum{}
%\pdfoutput=1 % Uncommented for upload to arXiv.org
%\CorrStatement{yes}  % For updates


%=================================================================
% Add packages and commands here. The following packages are loaded in our class file: fontenc, inputenc, calc, indentfirst, fancyhdr, graphicx, epstopdf, lastpage, ifthen, float, amsmath, amssymb, lineno, setspace, enumitem, mathpazo, booktabs, titlesec, etoolbox, tabto, xcolor, colortbl, soul, multirow, microtype, tikz, totcount, changepage, attrib, upgreek, array, tabularx, pbox, ragged2e, tocloft, marginnote, marginfix, enotez, amsthm, natbib, hyperref, cleveref, scrextend, url, geometry, newfloat, caption, draftwatermark, seqsplit
% cleveref: load \crefname definitions after \begin{document}

%=================================================================
% Please use the following mathematics environments: Theorem, Lemma, Corollary, Proposition, Characterization, Property, Problem, Example, ExamplesandDefinitions, Hypothesis, Remark, Definition, Notation, Assumption
%% For proofs, please use the proof environment (the amsthm package is loaded by the MDPI class).

%=================================================================
% Full title of the paper (Capitalized)
\Title{Title}

% MDPI internal command: Title for citation in the left column
\TitleCitation{Title}

% Author Orchid ID: enter ID or remove command
\newcommand{\orcidauthorA}{0000-0000-0000-000X} % Add \orcidA{} behind the author's name
%\newcommand{\orcidauthorB}{0000-0000-0000-000X} % Add \orcidB{} behind the author's name

% Authors, for the paper (add full first names)
\Author{Firstname Lastname $^{1,\dagger,\ddagger}$\orcidA{}, Firstname Lastname $^{2,\ddagger}$ and Jaime García-Mena $^{2,}$*}

%\longauthorlist{yes}

% MDPI internal command: Authors, for metadata in PDF
\AuthorNames{Firstname Lastname, Firstname Lastname and Firstname Lastname}

% MDPI internal command: Authors, for citation in the left column
\AuthorCitation{Lastname, F.; Lastname, F.; Lastname, F.}
% If this is a Chicago style journal: Lastname, Firstname, Firstname Lastname, and Firstname Lastname.

% Affiliations / Addresses (Add [1] after \address if there is only one affiliation.)
\address{%
$^{1}$ \quad Affiliation 1; e-mail@e-mail.com\\
$^{2}$ \quad Affiliation 2; e-mail@e-mail.com}

% Contact information of the corresponding author
\corres{Correspondence: e-mail@e-mail.com; Tel.: (optional; include country code; if there are multiple corresponding authors, add author initials) +xx-xxxx-xxx-xxxx (F.L.)}

% Current address and/or shared authorship
\firstnote{Current address: Affiliation.}  % Current address should not be the same as any items in the Affiliation section.
\secondnote{These authors contributed equally to this work.}
% The commands \thirdnote{} till \eighthnote{} are available for further notes

%\simplesumm{} % Simple summary

%\conference{} % An extended version of a conference paper

% Abstract (Do not insert blank lines, i.e. \\) 
\abstract{A single paragraph of about 200 words maximum. For research articles, abstracts should give a pertinent overview of the work. We strongly encourage authors to use the following style of structured abstracts, but without headings: (1) Background: place the question addressed in a broad context and highlight the purpose of the study; (2) Methods: describe briefly the main methods or treatments applied; (3) Results: summarize the article's main findings; (4) Conclusions: indicate the main conclusions or interpretations. The abstract should be an objective representation of the article, it must not contain results which are not presented and substantiated in the main text and should not exaggerate the main conclusions.}

% Keywords
\keyword{keyword 1; keyword 2; keyword 3 (List three to ten pertinent keywords specific to the article; yet reasonably common within the subject discipline.)} 

% The fields PACS, MSC, and JEL may be left empty or commented out if not applicable
%\PACS{J0101}
%\MSC{}
%\JEL{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Diversity
%\LSID{\url{http://}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Applied Sciences
%\featuredapplication{Authors are encouraged to provide a concise description of the specific application or a potential application of the work. This section is not mandatory.}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Data
%\dataset{DOI number or link to the deposited data set if the data set is published separately. If the data set shall be published as a supplement to this paper, this field will be filled by the journal editors. In this case, please submit the data set as a supplement.}
%\datasetlicense{License under which the data set is made available (CC0, CC-BY, CC-BY-SA, CC-BY-NC, etc.)}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Toxins
%\keycontribution{The breakthroughs or highlights of the manuscript. Authors can write one or two sentences to describe the most important part of the paper.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Encyclopedia
%\encyclopediadef{For entry manuscripts only: please provide a brief overview of the entry title instead of an abstract.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Advances in Respiratory Medicine and Smart Cities
%\addhighlights{yes}
%\renewcommand{\addhighlights}{%

%\noindent This is an obligatory section in “Advances in Respiratory Medicine'' and ``Smart Cities”, whose goal is to increase the discoverability and readability of the article via search engines and other scholars. Highlights should not be a copy of the abstract, but a simple text allowing the reader to quickly and simplified find out what the article is about and what can be cited from it. Each of these parts should be devoted up to 2~bullet points.\vspace{3pt}\\
%\textbf{What are the main findings?}
% \begin{itemize}[labelsep=2.5mm,topsep=-3pt]
% \item First bullet.
% \item Second bullet.
% \end{itemize}\vspace{3pt}
%\textbf{What is the implication of the main finding?}
% \begin{itemize}[labelsep=2.5mm,topsep=-3pt]
% \item First bullet.
% \item Second bullet.
% \end{itemize}
%}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% \setcounter{section}{-1} %% Remove this when starting to work on the template.
% \section{How to Use this Template}

% The template details the sections that can be used in a manuscript. Note that the order and names of article sections may differ from the requirements of the journal (e.g., the positioning of the Materials and Methods section). Please check the instructions on the authors' page of the journal to verify the correct order and names. For any questions, % please contact the editorial office of the journal or support@mdpi.com. For LaTeX-related questions please contact latex@mdpi.com.%\endnote{This is an endnote.} % To use endnotes, please un-comment \printendnotes below (before References). Only journal Laws uses \footnote.

% The order of the section titles is different for some journals. Please refer to the "Instructions for Authors” on the journal homepage.

\section{Introduction}

The introduction should briefly place the study in a broad context and highlight why it is important. It should define the purpose of the work and its significance. The current state of the research field should be reviewed carefully and key publications cited. Please highlight controversial and diverging hypotheses when necessary. Finally, briefly mention the main aim of the work and highlight the principal conclusions. As far as possible, please keep the introduction comprehensible to scientists outside your particular field of research. Citing a journal paper \cite{ref-journal}. Now citing a book reference \cite{ref-book1,ref-book2} or other reference types \cite{ref-unpublish,ref-communication,ref-proceeding}. Please use the command \citep{ref-thesis,ref-url} for the following MDPI journals, which use author--date citation: Administrative Sciences, Arts, Econometrics, Economies, Genealogy, Humanities, IJFS, Journal of Intelligence, Journalism and Media, JRFM, Languages, Laws, Religions, Risks, Social Sciences, Literature.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Materials and Methods}

\subsection{Experimental Design}
This study recruited Mexican women aged 18--45 years from gynecological consultations at ABC Medical Center. Participants were divided into two groups: cases and controls. Cases were women with recurrent vulvovaginal candidiasis (RVVC), defined as at least four episodes in the past 12 months, while controls were healthy women attending routine annual Pap smear consultations without a history of recurrent vaginitis. 

Inclusion criteria for both groups required abstinence from sexual activity for 48 hours, no menstruation at the time of sample collection, no antibiotic use in the previous three months, and no diagnosis of diabetes mellitus or immunosuppressive disorders. Cases were additionally required to have a history of RVVC, while controls were limited to a history of one or fewer vaginitis episodes over their lifetime. Participants provided informed consent after receiving a detailed explanation of the study.

Exclusion criteria included conditions contrary to the inclusion parameters, such as menstruation during sample collection, recent antibiotic use, or declining to participate. Controls were excluded if they had symptoms of vaginitis within the past six months or a history of more than one vaginitis episode diagnosed as vulvovaginal candidiasis. Cases were excluded if they did not meet the threshold for RVVC episodes or if they were outside the specified age range.

\subsection{Vaginal Swab Sampling}
Participants were positioned on a gynecological examination table with stirrups, and a vaginal speculum was inserted for sample collection. Cervicovaginal mucus was collected using Catch-All™ Sample Collection Swabs (Epicentre, Illumina) from the vaginal fornix. The vaginal pH was measured using Hydrion pH test strips (MicroEssential). The swab was then placed in a 2 mL tube containing sterile PBS solution, trimmed to fit, and stored at 4$^\circ$ C until DNA extraction.

\subsection{DNA Extraction from Vaginal Swab Samples}
DNA was extracted from vaginal swab samples using the GeneAll Exgene™ Stool SV Kit (GeneAll). Samples were first centrifuged for 5 minutes at 6000 rpm, and the supernatant was discarded. The pellet was resuspended in 1 mL of PBS buffer, vortexed for 1 minute, and incubated at room temperature for 30 seconds. After centrifugation at maximum speed for 2 minutes, the supernatant was discarded. The pellet was resuspended in 1.3 mL of Buffer FL, incubated for 5 minutes at room temperature, and centrifuged at $\geq$10,000 $\times g$ for 5 minutes. The supernatant was transferred to an EzPass™ filter column, centrifuged, and eluted into a clean 1.5 mL tube with 100 µL of Buffer EB. After further purification steps involving Buffer PB and Buffer NW, the final DNA elution was performed with 50 µL of Buffer EB. DNA integrity and concentration were verified using a Nanodrop 2000 spectrophotometer (Thermo Scientific) and 0.5\% agarose gel electrophoresis.

\subsection{Amplification of 16S Ribosomal Gene (V3 and V4 Regions)}
The presence of bacterial DNA in vaginal swab samples was confirmed by amplifying the V3 and V4 regions of the 16S ribosomal RNA gene using polymerase chain reaction (PCR). Reaction conditions and primers followed the protocols detailed in Tables 3 and 4, with a final reaction volume of 50 µL per sample.

\emph{Escherichia coli} SK10019 was used as a positive control for bacterial DNA, and a reaction mix without template DNA served as the negative control. Amplifications were carried out using a GeneAmp\textregistered~PCR System 2700 thermocycler (Applied Biosystems). PCR products were resolved on 2\% agarose gels stained with Midori Green Advanced dye, using TBE buffer. A molecular weight marker of 100 bp (Fermentas\textregistered) was included. Electrophoresis was conducted at 90 V for 50 minutes, and gels were imaged using a Molecular Imager\textregistered~Gel Doc\texttrademark~XR system (Bio-Rad).

\subsection{Library Preparation for the V3 Region of the 16S rRNA Gene}
Bacterial DNA presence was confirmed in vaginal samples, followed by library preparation targeting the V3 region of the 16S rRNA gene. A unique barcode sequence and sequencing adapters were incorporated using PCR under conditions detailed in Tables 5 and 6. Amplicon quality was verified via 2.0\% agarose gel electrophoresis. Products of $\sim$281 bp were identified, and large-scale PCR was performed to increase product yield.

Equal concentrations of PCR products were pooled and further purified from preparative agarose gels using the Wizard\textregistered~SV Gel and PCR Clean-Up System (Promega). Final amplicon size ($\sim$281 bp) and DNA concentration were validated by analytical 2.0\% agarose gel electrophoresis.

\subsection{Amplification of the V5 Region of the 18S rRNA Gene}
To detect yeast DNA, analytical PCR targeting the V5 region of the 18S rRNA gene was performed using high-fidelity Takara Ex Taq polymerase under conditions described in Tables 7 and 8. \emph{Saccharomyces cerevisiae} S2886 served as a positive control, and a no-template reaction served as a negative control. Products were analyzed via 2.0\% agarose gel electrophoresis.

\subsection{Species Diagnosis of \emph{Candida} via PCR Amplification}
To determine the etiology of recurrent vulvovaginal candidiasis (RVVC) in case subjects and detect the presence of \emph{Candida} in control subjects, PCR reactions were performed using species-specific primers for \emph{C. albicans} and \emph{C. glabrata}. These species were selected based on epidemiological prevalence, and primers for \emph{Saccharomyces cerevisiae} were included due to its potential role in vulvovaginitis, which clinically resembles \emph{Candida} infections. 

Primers for \emph{C. albicans} and \emph{C. glabrata} targeted the Internal Transcribed Spacer (ITS) region, as described by Luo and Mitchell (2002). \emph{S. cerevisiae} primers targeted the MEX67 gene, involved in mRNA nuclear export, following methods described by Muir et al. (2011). PCR reactions used high-fidelity Takara Ex Taq polymerase with conditions and primer sequences detailed in Tables 10--13. Positive controls included \emph{C. albicans} ATCC, \emph{C. glabrata} CBS138, and \emph{S. cerevisiae} S2886, while negative controls used reaction mixtures without DNA templates. 

Amplicons were resolved via 2.0\% agarose gel electrophoresis stained with Midori Green Advanced dye, and products were visualized using the Molecular Imager\textregistered~Gel Doc\texttrademark~XR system (Bio-Rad). Amplicon sizes were confirmed against a 100 bp molecular weight marker. 

PCR products were cloned using the GeneJET\texttrademark~PCR Cloning Kit (Thermo Fisher), and plasmid DNA was extracted for capillary sequencing with pJET1 primers. Resulting sequences were analyzed using VECTOR NTI Advance\texttrademark~and BLAST alignments to verify species-specific diagnostic accuracy. Figures 14--16 depict the pJET1.2blunt vector and the cloned sequences of each amplified product.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Results}

\subsection{Distinct Microbial Profiles and Candida Prevalence in Vaginosis Cases vs. Controls}

In this study, 38 controls and 57 cases met the inclusion criteria, of which 25 controls and 48 cases were successfully sequenced (Table~\ref{stab:seq_sum}). No significant differences were observed between groups regarding age, age at menarche, age of sexual life start, or vaginal pH, with other variables showing similar distributions between the groups (Figure~\ref{sfig:qqplot}). Notably, a subset of participants had used antibiotics (30.95\% of cases and 43.86\% of controls), and this factor was accounted for in subsequent analyses. Additionally, qPCR detection revealed significant differences in the prevalence of \textit{Candida albicans} (90\% in cases vs. 36.17\% in controls) and \textit{Candida glabrata} (0\% in cases vs. 97.87\% in controls) based on two-proportion z-tests (Table~\ref{tab:metadata}, Figure~\ref{fig:metadata}).

\begin{table}[H]
    \centering
    \caption{Quantified variables in the studied subjects.}
    \label{tab:metadata}
    \begin{tabular}{@{}lccc}
    	\toprule
    	\textbf{Variable} & \textbf{Control} & \textbf{Cases} & \textbf{\textit{p}} \\
        \midrule
        Number & 38 & 57 & \\
        Age & 31.92 (±7.67) [38] & 35.31 (±10.22) [51] & 0.08 \\
        Menarche Age & 12.53 (±1.59) [38] & 12.64 (±1.44) [51] & 0.71 \\
        Age Start Sexual Life & 19.67 (±3.00) [36] & 18.62 (±2.89) [48] & 0.11 \\
        \midrule
        \textbf{Contraceptive Method} & & & \\
        None & 14/32 (43.75\%) & 29/57 (50.88\%) & \\
        Tubal ligation & 1/32 (3.12\%) & 8/57 (14.03\%) & \\
        Condom & 6/32 (18.75\%) & 10/57 (17.54\%) & \\
        Intrauterine device & 2/32 (6.25\%) & 6/57 (10.53\%) & \\
        Oral & 7/32 (21.88\%) & 4/57 (7.02\%) & \\
        Implant & 2/32 (6.25\%) & 0/57 (0.00\%) & \\
        \midrule
        \textbf{Menstrual Cycle Phase} & & & \\
        Luteal & 19/32 (59.38\%) & 32/55 (58.18\%) & \\
        Follicular & 11/32 (34.38\%) & 17/55 (30.91\%) & \\
        Ovulation & 2/32 (6.25\%) & 6/55 (10.91\%) & \\
        \textbf{Sex Partners} & & & \\
        1 & 15/32 (46.88\%) & 21/54 (38.89\%) & \\
        $\geq$2 & 17/32 (53.13\%) & 33/54 (61.11\%) & \\
        \midrule
        \textbf{Gestations} & & & \\
        0 & 17/31 (54.84\%) & 17/52 (32.69\%) & \\
        1-2 & 7/31 (22.58\%) & 18/52 (34.62\%) & \\
        $\geq$3 & 7/31 (22.58\%) & 17/52 (32.69\%) & \\
        \midrule
        \textbf{Delivers} & & & \\
        0 & 22/31 (70.97\%) & 32/52 (61.54\%) & \\
        1-2 & 4/31 (12.90\%) & 13/52 (25.00\%) & \\
        $\geq$3 & 5/31 (16.13\%) & 7/52 (13.46\%) & \\
        \midrule
        \textbf{Cesarean sections} & & & \\
        0 & 25/31 (80.65\%) & 30/52 (57.69\%) & \\
        1-2 & 5/31 (16.13\%) & 21/52 (40.38\%) & \\
        $\geq$3 & 1/31 (3.23\%) & 1/52 (1.92\%) & \\
        \midrule
        \textbf{Abortions} & & & \\
        0 & 29/31 (93.54\%) & 39/52 (75.00\%) & \\
        1-2 & 1/31 (3.22\%) & 12/52 (23.07\%) & \\
        $\geq$3 & 1/3 (3.22\%) & 1/52 (1.92\%) & \\
        \midrule
        Vaginal pH & & & \\
        mean pH & 4.48 ± 0.57 [29] & 4.78 ± 1.25 [43] & 0.18 \\
        $\leq$4 & 14/25 (56.00\%) & 17/40 (42.50\%) & \\
        $\geq$5 & 11/25 (44.00\%) & 23/40 (57.50\%) & \\
        \midrule
        Smoking & 8/42 (19.05\%) & 19/58 (33.33\%) & \\
        Antibiotics & 13/42 (30.95\%) & 25/57 (43.86\%) & \\
        Antifungal & 0/33 (0.00\%) & 5/57 (8.77\%) & \\
        \midrule
        Vulvovaginal Candidiasis episodes & & & \\
        0 & 31/31 (100.00\%) & 0/52 (0.00\%) & \\
        1-2 & 0/31 (0.00\%) & 38/52 (73.08\%) & \\
        $\geq$3 & 0/31 (0.00\%) & 14/52 (26.92\%) & \\
        \midrule
        \textbf{qPCR \textit{Candida} detection} & & & \\
        \textit{Candida albicans} & 27/30 (90.00\%) & 17/47 (36.17\%) & 9.93x10$^{-6}$ \\
        \textit{Candida glabrata} & 0/30 (0.00\%) & 46/47 (97.87\%) & <2.2x10$^{-16}$ \\
        \textit{Saccharomyces cerevisiae} & 27/30 (90.00\%) & 45/47 (95.74\%) & 0.6 \\
        \bottomrule
    \end{tabular}
    \caption*{ Continuous data are represented as mean ± standard deviation, number individuals with available data is enclosed in square brackets. Categorical data are represented as proportions, percentages are enclosed in parentheses. For age, menarche age, start of sexual life and pH, student's \textit{t}-test was applied. For qPCR detection results, a two-proportion \textit{z}-test was used. }
\end{table}
    
\begin{figure}[H]
	\centering
	\includegraphics[width = \textwidth]{figures/metadata.png}
	\caption{Main variables driving the study. \textbf{A.} Histogram depicting the age distribution between control and case groups. No statistical difference was found according to Student's t-test.\textbf{B.} Barplot showing antibiotic intake in women. \textbf{C.} Barplot showing proportion of control and case groups with three different Fungi detected by qPCR.}
	\label{fig:metadata}
\end{figure}   


\subsection{Antibiotic Intake and Patient Condition Have a Small Effect on Beta Diversity}

Sequencing data were processed as outlined in the Methods section. For samples with low sequencing depth, re-sequencing was performed, and the resulting reads were combined by summation (Figure~\ref{sfig:sample_sums}). 

To investigate the impact of antibiotic use and patient condition on global bacterial diversity, both alpha and beta diversity metrics were analyzed. While alpha diversity appeared unaffected by these factors, beta diversity showed significant associations based on ADONIS tests. Weighted beta diversity was influenced primarily by patient condition (\textit{p} = 0.042), whereas unweighted beta diversity was significantly associated with both patient condition (\textit{p} = 0.012) and antibiotic use (\textit{p} = 0.018). Despite these findings, the low $R^2$ values indicate that the observed effects are small (Figure~\ref{fig:diversity}). 

Additionally, sequencing depth was examined in relation to antibiotic consumption to determine whether it contributed to a reduction in read counts; however, no evidence of such an effect was found (Figure~{\ref{sfig:counts_ant}}).

\begin{figure}[H]
    \centering
    \includegraphics[width = \textwidth]{figures/diversity.png}
    \caption{ Diversity metrics for stratified groups, considering antibiotic intake and condition (case vs. control). \textbf{A.} Violin plot depicting alpha diversity metrics (Observed number of ASVs, Shannon, Simpson and Fisher) for the groups. \textbf{B.} Scatter plot showing Non-Metric Multidimensional Scaling (NMDS) for weighted Unifrac distance for beta diversity assesing in the groups. ADONIS was applied in order to account for the variance explained by Condition (\textit{p} = 0.036, $R^{2}$ = 0.05) and Antibiotic intake (\textit{p} = 0.39, $R^{2}$ = 0.01). \textbf{C.} Scatter plot showing Non-Metric Multidimensional Scaling (NMDS) for unweighted Unifrac distance for beta diversity assesing in the groups. ADONIS was applied in order to account for the variance explained by Condition (\textit{p} = 0.012, $R^{2}$ = 0.03) and Antibiotic intake (\textit{p} = 0.018, $R^{2}$ = 0.03).}
	\label{fig:diversity}
\end{figure}

\subsection{Vaginal Microbiota Composition Reflects Patient Condition and Antibiotic Use}

The composition of the vaginal microbiota was analyzed at both the phylum and genus levels to identify changes in key bacterial taxa across samples. Hierarchical clustering using weighted beta diversity was employed to compare taxa and assess similarities among bacterial communities in the groups.

Samples were first grouped by antibiotic intake and subsequently by patient condition. Significant alterations were observed in the \textit{Firmicutes} ratio, particularly between cases and controls. Cases without antibiotic use exhibited the highest abundance of \textit{Bacteroidota} and \textit{Actinobacteriota}, whereas controls who had taken antibiotics were dominated by the \textit{Firmicutes\_D} phylum.

At the genus level, \textit{Firmicutes\_D} was predominantly represented by \textit{Lactobacillus}, which also dominated controls with antibiotic use. In contrast, cases without antibiotic intake showed a higher prevalence of \textit{Bifidobacterium}, while  \textit{Bacteroides\_H} was found only in controls who had not taken antibiotics. Notably, \textit{Fannyhessea} was characteristic of cases without antibiotic use, whereas \textit{Sneathia} was specific to cases with antibiotic use.

Additionally, a marked increase in \textit{Prevotella} was observed in cases where \textit{Lactobacillus} was replaced by a diverse array of low-abundance taxa (<2\%, grouped as "Other"). These findings highlight the dynamic shifts in microbial composition associated with antibiotic use and patient condition (Figure~{\ref{fig:relab}}).

\begin{figure}[H]
    \centering
    \includegraphics[width = \textwidth]{figures/relab.png}
    \caption{ Relative abundance for stratified groups, considering antibiotic intake and condition (case vs. control). \textbf{A.} Phylum relative abundance as percentages. \textbf{B.} Genus relative abundance as percentages.}
	\label{fig:relab}
    \end{figure}

\subsection{\textit{Lactobacillus} and Other Key Bacterial Biomarkers Are Associated with a Healthy Vaginal Microbiota}

To identify bacterial biomarkers associated with vaginosis, we performed differential abundance analyses while accounting for antibiotic intake. Two approaches were used: ALDEx2, with the formula $y \sim$ antibiotic intake + condition, and linear discriminant analysis (LEfSe), using condition as the class variable and antibiotic intake as the subclass.

The results from ALDEx2 are presented in Figure~\ref{fig:differential}A, with corresponding square root-transformed relative abundance values shown in Figure~\ref{fig:differential}B. In the control group, \textit{Lactobacillus} and an unassigned taxon were prominent. Interestingly, two distinct ASVs classified as \textit{Escherichia\_710834} were identified in both the control and case groups.

LEfSe results are shown in Figure~\ref{fig:differential}C, with the corresponding square root-transformed relative abundance values in Figure~\ref{fig:differential}D. These findings corroborated the ALDEx2 results for \textit{Lactobacillus}. Additionally, \textit{Prevotella}, \textit{Limosilactobacillus}, \textit{Streptococcus}, and \textit{Dialister} were predominant in controls, whereas two distinct \textit{Escherichia\_710834} ASVs, unassigned taxa, and \textit{Cutibacterium} were characteristic of cases.

\begin{figure}[H]
    \centering
    \includegraphics[width = \textwidth]{figures/Differential.jpg}
    \caption{ Important biomarker bacteria considering antibiotic intake and condition. A and B. Using ALDEx2 ( y ~ antibiotic intake + condition). C and D Using LEFSE (Class = condidion, Subclass = antibiotic intake). Only features with p-values which passed FDR correction were included in the graphs. \textbf{A.} Barplot showing ALDEx2 results for condition (case vs. control). X-axis shows the effect, Y-axis shows the genus. Negative values correspond to control, positive values correspond to cases.
    \textbf{B}. Barplot of mean sqrt of relative abundance of taxa found by ALDEx2. X-axis shows mean sqrt of relative abundance, Y-axis shows genus. \textbf{C.} Barplot showing LEFSE results for condition (case vs. control). X-axis shows the LDA scores, Y-axis shows the genus. Negative values correspond to control, positive values correspond to cases. \textbf{D}. Barplot of mean sqrt of relative abundance of taxa found by LEFSE. X-axis shows mean sqrt of relative abundance, Y-axis shows genus.}
	\label{fig:differential}
    \end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\section{Discussion}

The aim of this work was to characterize the vaginal microbiota of women suffering from RVVC. We found no differences between cases and control in women associated metadata. For instance, no differences in vaginal pH were consistent with the disease diagnosis \cite{@sobelVulvovaginalCandidosis2007}. The age prevalence in women is also consistent with literature which indicates that most of episodes occur between 19 to 35 years with lower prevalence rates after 50 years \cite{@blosteinRecurrentVulvovaginalCandidiasis2017}, though we observed a broader range between 20 to 50 years, with most of the cases being around 35. Additionally, a moderate association with RVVC incidence and new sexual partners has been reported \cite{@yanoCurrentPatientPerspectives2019}, in our work, there is a tendency of cases having more sexual partners, but our results are not conclusive in that matter.

In this work, we detected by qPCR, \textit{Candida albicans} exclusive to RVVC cases, while \textit{Candida glabrata} was found in higher proportion in the control group. Interestingly, some literature mentions \textit{Candida albicans} to be the major cause of infection, with 80-95\% of isolates identified as this species \cite{@sobelVulvovaginalCandidosis2007a}, though somo other species such as \textit{Candida glabrata}, \textit{Candida tropicalis}, and \textit{Candida parapsilosis} are now frequently identified as human pathogens \cite{@silvaCandidaGlabrataCandida2012}. Moreover, \textit{Candida} spp. have been previously isolated for healthy women \cite{@goldacreVaginalMicrobialFlora1979; @chowVaginalColonizationEscherichia1986}, and it has been reported that \textit{Candida} infections are in fact common and in most of the cases occurs in healthy individuals \cite{@gowImportanceCandidaAlbicans2012}, so there might be an effect of the vaginal bacteria in our patients, which will be explored further.

It was also interesting, for the scope of this study, that a significant portion of both group took antibiotics during this study, as it has been observed previously that intake of broad-spectrum antibiotics might reduce the gut microbiota diversity \cite{@dubourgCulturomicsPyrosequencingEvidence2014}. For this reason, we had to include antibiotics intake into account when evaluating the bacterial profile.

We found no major differences in alpha diversity between cases and controls in our work. However, a small effect was observed due to both antibiotic intake and RVVC. As mentioned before, is expected that antibiotics have an impact in diversity, also in the context of vaginal microbiota, long-term might develop resistance and cause recurrent infections \cite{@lev-sagieVaginalMicrobiomeTransplantation2019}. Overall, the diversity in all groups was low, which is expected in the case of vaginal microbiota \cite{@sunVulvovaginalCandidiasisVaginal2023}. Previous reports evaluating microbial diversity during RVVC condition found major differences in alpha and beta diversity, with lower diversity in the cases \cite{@ceccaraniDiversityVaginalMicrobiome2019; @liuDiverseVaginalMicrobiomes2013}.



Clostridioides difficile (formerly known as Clostridium difficile) infection is an example of a disease brought about directly through antibiotic disruption of the gut microbiota /\cite{@theriotAntibioticinducedShiftsMouse2014}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Conclusions}

This section is not mandatory, but can be added to the manuscript if the discussion is unusually long or complex.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Patents}

This section is not mandatory, but may be added if there are patents resulting from the work reported in this manuscript.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\vspace{6pt} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% optional
%\supplementary{The following supporting information can be downloaded at:  \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title.}

% Only for journal Methods and Protocols:
% If you wish to submit a video article, please do so with any other supplementary material.
% \supplementary{The following supporting information can be downloaded at: \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title. A supporting video article is available at doi: link.}

% Only for journal Hardware:
% If you wish to submit a video article, please do so with any other supplementary material.
% \supplementary{The following supporting information can be downloaded at: \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title.\vspace{6pt}\\
%\begin{tabularx}{\textwidth}{lll}
%\toprule
%\textbf{Name} & \textbf{Type} & \textbf{Description} \\
%\midrule
%S1 & Python script (.py) & Script of python source code used in XX \\
%S2 & Text (.txt) & Script of modelling code used to make Figure X \\
%S3 & Text (.txt) & Raw data from experiment X \\
%S4 & Video (.mp4) & Video demonstrating the hardware in use \\
%... & ... & ... \\
%\bottomrule
%\end{tabularx}
%}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\authorcontributions{For research articles with several authors, a short paragraph specifying their individual contributions must be provided. The following statements should be used ``Conceptualization, X.X. and Y.Y.; methodology, X.X.; software, X.X.; validation, X.X., Y.Y. and Z.Z.; formal analysis, X.X.; investigation, X.X.; resources, X.X.; data curation, X.X.; writing---original draft preparation, X.X.; writing---review and editing, X.X.; visualization, X.X.; supervision, X.X.; project administration, X.X.; funding acquisition, Y.Y. All authors have read and agreed to the published version of the manuscript.'', please turn to the  \href{http://img.mdpi.org/data/contributor-role-instruction.pdf}{CRediT taxonomy} for the term explanation. Authorship must be limited to those who have contributed substantially to the work~reported.}

\funding{Please add: ``This research received no external funding'' or ``This research was funded by NAME OF FUNDER grant number XXX.'' and  and ``The APC was funded by XXX''. Check carefully that the details given are accurate and use the standard spelling of funding agency names at \url{https://search.crossref.org/funding}, any errors may affect your future funding.}

\institutionalreview{In this section, you should add the Institutional Review Board Statement and approval number, if relevant to your study. You might choose to exclude this statement if the study did not require ethical approval. Please note that the Editorial Office might ask you for further information. Please add “The study was conducted in accordance with the Declaration of Helsinki, and approved by the Institutional Review Board (or Ethics Committee) of NAME OF INSTITUTE (protocol code XXX and date of approval).” for studies involving humans. OR “The animal study protocol was approved by the Institutional Review Board (or Ethics Committee) of NAME OF INSTITUTE (protocol code XXX and date of approval).” for studies involving animals. OR “Ethical review and approval were waived for this study due to REASON (please provide a detailed justification).” OR “Not applicable” for studies not involving humans or animals.}

\informedconsent{Any research article describing a study involving humans should contain this statement. Please add ``Informed consent was obtained from all subjects involved in the study.'' OR ``Patient consent was waived due to REASON (please provide a detailed justification).'' OR ``Not applicable'' for studies not involving humans. You might also choose to exclude this statement if the study did not involve humans.

Written informed consent for publication must be obtained from participating patients who can be identified (including by the patients themselves). Please state ``Written informed consent has been obtained from the patient(s) to publish this paper'' if applicable.}

\dataavailability{We encourage all authors of articles published in MDPI journals to share their research data. In this section, please provide details regarding where data supporting reported results can be found, including links to publicly archived datasets analyzed or generated during the study. Where no new data were created, or where data is unavailable due to privacy or ethical restrictions, a statement is still required. Suggested Data Availability Statements are available in section ``MDPI Research Data Policies'' at \url{https://www.mdpi.com/ethics}.} 

% Only for journal Nursing Reports
%\publicinvolvement{Please describe how the public (patients, consumers, carers) were involved in the research. Consider reporting against the GRIPP2 (Guidance for Reporting Involvement of Patients and the Public) checklist. If the public were not involved in any aspect of the research add: ``No public involvement in any aspect of this research''.}

% Only for journal Nursing Reports
%\guidelinesstandards{Please add a statement indicating which reporting guideline was used when drafting the report. For example, ``This manuscript was drafted against the XXX (the full name of reporting guidelines and citation) for XXX (type of research) research''. A complete list of reporting guidelines can be accessed via the equator network: \url{https://www.equator-network.org/}.}

% Only for journal Nursing Reports
%\useofartificialintelligence{Please describe in detail any and all uses of artificial intelligence (AI) or AI-assisted tools used in the preparation of the manuscript. This may include, but is not limited to, language translation, language editing and grammar, or generating text. Alternatively, please state that “AI or AI-assisted tools were not used in drafting any aspect of this manuscript”.}

\acknowledgments{In this section you can acknowledge any support given which is not covered by the author contribution or funding sections. This may include administrative and technical support, or donations in kind (e.g., materials used for experiments).}

\conflictsofinterest{Declare conflicts of interest or state ``The authors declare no conflicts of interest.'' Authors must identify and declare any personal circumstances or interest that may be perceived as inappropriately influencing the representation or interpretation of reported research results. Any role of the funders in the design of the study; in the collection, analyses or interpretation of data; in the writing of the manuscript; or in the decision to publish the results must be declared in this section. If there is no role, please state ``The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript; or in the decision to publish the results''.} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Optional

%% Only for journal Encyclopedia
%\entrylink{The Link to this entry published on the encyclopedia platform.}

\abbreviations{Abbreviations}{
The following abbreviations are used in this manuscript:\\

\noindent 
\begin{tabular}{@{}ll}
MDPI & Multidisciplinary Digital Publishing Institute\\
DOAJ & Directory of open access journals\\
TLA & Three letter acronym\\
LD & Linear dichroism
\end{tabular}
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Optional
\appendixtitles{yes} % Leave argument "no" if all appendix headings stay EMPTY (then no dot is printed after "Appendix A"). If the appendix sections contain a heading then change the argument to "yes".
\appendixstart
\appendix
\section[\appendixname~\thesection]{Supplementary Tables}

\begin{table}[H] 
	\caption{Sequencing summary. \label{stab:seq_sum}}
	\centering
	\begin{tabular}{l|cc}
		\toprule
		\textbf{Parameter} & \textbf{Control (n = 27)} & \textbf{Case (n = 54)} \\
		\midrule
		Total reads       & 2,012,838  & 3,758,769  \\
		mean              & 74,549.56  & 69,606.83  \\
		sd                & 69,222.23  & 53,048.33  \\
		median            & 56,828.00  & 55,813.50  \\
		min               & 5,668      & 5,907      \\
		max               & 259,685    & 200,426    \\
		\bottomrule
	\end{tabular}
\end{table}
	

\section[\appendixname~\thesection]{Supplementary Figures}

\begin{figure}[H]
	\centering
	\includegraphics[width = 0.6\textwidth]{figures/qqplot.png}
	\caption{Normal Q-Q plot showing normal distribution of age in the studied women. Age was normally distributed according to Shapiro-Wilk test (p-value < 0.05).}
	\label{sfig:qqplot}
\end{figure}

\begin{figure}[H]
	\centering
	\includegraphics[width = 0.9\textwidth]{figures/sample_sums_bm.png}
	\caption{Barplot showing sequencing depth for each sample. Samples highlighted in red were sequenced twice and merged due they exhibited low sequencing depth.}
	\label{sfig:sample_sums}
\end{figure}

\begin{figure}[H]
	\centering
	\includegraphics[width = 0.6\textwidth]{figures/counts_ant.png}
	\caption{Barplot showing sample counts and antibiotic intake in both case and control groups.}
	\label{sfig:counts_ant}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{adjustwidth}{-\extralength}{0cm}
%\printendnotes[custom] % Un-comment to print a list of endnotes

\reftitle{References}

% Please provide either the correct journal abbreviation (e.g. according to the “List of Title Word Abbreviations” http://www.issn.org/services/online-services/access-to-the-ltwa/) or the full name of the journal.
% Citations and References in Supplementary files are permitted provided that they also appear in the reference list here. 

%=====================================
% References, variant A: external bibliography
%=====================================
%\bibliography{your_external_BibTeX_file}

%=====================================
% References, variant B: internal bibliography
%=====================================
\begin{thebibliography}{999}
% Reference 1
\bibitem[Author1(year)]{ref-journal}
Author~1, T. The title of the cited article. {\em Journal Abbreviation} {\bf 2008}, {\em 10}, 142--149.
% Reference 2
\bibitem[Author2(year)]{ref-book1}
Author~2, L. The title of the cited contribution. In {\em The Book Title}; Editor 1, F., Editor 2, A., Eds.; Publishing House: City, Country, 2007; pp. 32--58.
% Reference 3
\bibitem[Author3(year)]{ref-book2}
Author 1, A.; Author 2, B. \textit{Book Title}, 3rd ed.; Publisher: Publisher Location, Country, 2008; pp. 154--196.
% Reference 4
\bibitem[Author4(year)]{ref-unpublish}
Author 1, A.B.; Author 2, C. Title of Unpublished Work. \textit{Abbreviated Journal Name} year, \textit{phrase indicating stage of publication (submitted; accepted; in press)}.
% Reference 5
\bibitem[Author5(year)]{ref-communication}
Author 1, A.B. (University, City, State, Country); Author 2, C. (Institute, City, State, Country). Personal communication, 2012.
% Reference 6
\bibitem[Author6(year)]{ref-proceeding}
Author 1, A.B.; Author 2, C.D.; Author 3, E.F. Title of presentation. In Proceedings of the Name of the Conference, Location of Conference, Country, Date of Conference (Day Month Year); Abstract Number (optional), Pagination (optional).
% Reference 7
\bibitem[Author7(year)]{ref-thesis}
Author 1, A.B. Title of Thesis. Level of Thesis, Degree-Granting University, Location of University, Date of Completion.
% Reference 8
\bibitem[Author8(year)]{ref-url}
Title of Site. Available online: URL (accessed on Day Month Year).
\end{thebibliography}

% If authors have biography, please use the format below
%\section*{Short Biography of Authors}
%\bio
%{\raisebox{-0.35cm}{\includegraphics[width=3.5cm,height=5.3cm,clip,keepaspectratio]{Definitions/author1.pdf}}}
%{\textbf{Firstname Lastname} Biography of first author}
%
%\bio
%{\raisebox{-0.35cm}{\includegraphics[width=3.5cm,height=5.3cm,clip,keepaspectratio]{Definitions/author2.jpg}}}
%{\textbf{Firstname Lastname} Biography of second author}

% For the MDPI journals use author-date citation, please follow the formatting guidelines on http://www.mdpi.com/authors/references
% To cite two works by the same author: \citeauthor{ref-journal-1a} (\citeyear{ref-journal-1a}, \citeyear{ref-journal-1b}). This produces: Whittaker (1967, 1975)
% To cite two works by the same author with specific pages: \citeauthor{ref-journal-3a} (\citeyear{ref-journal-3a}, p. 328; \citeyear{ref-journal-3b}, p.475). This produces: Wong (1999, p. 328; 2000, p. 475)

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% for journal Sci
%\reviewreports{\\
%Reviewer 1 comments and authors’ response\\
%Reviewer 2 comments and authors’ response\\
%Reviewer 3 comments and authors’ response
%}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\PublishersNote{}
\end{adjustwidth}
\end{document}

